comps/llms/text-generation/ray_serve/launch_ray_service.sh

```diff
@@ -5,8 +5,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Set default values
-default_port=8080
-default_model="meta-llama/Llama-2-7b-chat-hf"
+default_port=8008
```

Comment:
When changing the default_port, don't forget to update the example in the README.

Reply:
The port in readme is originally 8008, I just update code according to readme.